tests/workflows/components/workflows.libsonnet

```diff
@@ -67,20 +59,9 @@
       local k8sPy = srcDir;
       local kubeflowPy = srcRootDir + "/kubeflow/testing/py";
 
-      local project = params.project;
-      // GKE cluster to use
-      // We need to truncate the cluster to no more than 40 characters because
-      // cluster names can be a max of 40 characters.
-      // We expect the suffix of the cluster name to be unique salt.
-      // We prepend a z because cluster name must start with an alphanumeric character
-      // and if we cut the prefix we might end up starting with "-" or other invalid
-      // character for first character.
-      local cluster =
-        if std.length(name) > 40 then
-          "z" + std.substr(name, std.length(name) - 39, 39)
-        else
-          name;
-      local zone = params.zone;
+      // EKS cluster name, better to be meaningful to trace back to prow job
+      // Maximum length of cluster name is 100. We set to 80 as maximum here and truncate
```

Comment:
Where did the truncation happen? It seems the above logic has been removed.

Reply:
Yeah I forgot to remove the comment

Reply:
Yeah I forgot to remove the comment